### ISSUE TITLE
Review VM configuration is not visible when creating workspace

### ISSUE THREAD

**Issue by tanya-borisova** [microsoft/AzureTRE#2830]

**Comment by SvenAelterman** [comment]
Agreed with this one. It's relatively minor, but when deploying a workspace for the first time, seeing checkboxes and textboxes for items you can't successfully configure anyway causes confusion and despair :)

**Comment by martinpeck** [comment]
marked with triage as bug needs description
